ebean-api/src/main/java/io/ebean/DB.java

```diff
@@ -251,7 +250,7 @@ public static Transaction currentTransaction() {
    * </ul>
    */
   public static void flush() {
-    currentTransaction().flush();
+    getDefault().flush();
```

Comment:
Delegating to DefaultServer.flush, which may throw a "no currentTransaction" exception.